.gitignore

```diff
@@ -55,3 +55,12 @@ public/sitemap*
 # Ignore local configuration
 .env.development
 .env.test
+
+# Ignore developer cruft
+**/.DS_Store
```

Comment:
@smith, I've reverted that commit and re-pushed the branch.  Let me know if I should rebase that commit out of the branch, instead of having those two commits muddy the history.